README.md

```diff
@@ -153,7 +134,7 @@ var flicking = new eg.Flicking("#el", { collectStatistics: false });
 ## 🌐 Supported Browsers
 |<img width="20" src="https://simpleicons.org/icons/internetexplorer.svg" alt="IE" />|<img width="20" src="https://simpleicons.org/icons/googlechrome.svg" alt="Chrome" />|<img width="20" src="https://simpleicons.org/icons/firefoxbrowser.svg" alt="Firefox" />|<img width="20" src="https://simpleicons.org/icons/safari.svg" alt="Safari" />|<img width="20" src="https://simpleicons.org/icons/apple.svg" alt="iOS" />|<img width="20" src="https://simpleicons.org/icons/android.svg" alt="Android" />|
 |:---:|:---:|:---:|:---:|:---:|:---:|
-|10+|Latest|Latest|Latest|7+|4+|
+|9+(With polyfill)|Latest|Latest|Latest|7+|4+|
```

Comment:
I think that In [our polyfill policy](https://oss.navercorp.com/egjs/work/issues/575#issuecomment-4277333), it seems that we need to define which polyfill to put.

Reply:
Good, I think it only needs Promise polyfill but should be tested.

Reply:
It would be nice if an explanation of what polyfill to add was added.

Reply:
I've tested it and only es6 Promise polyfill was needed to run in IE9+